### DISCUSSION TITLE
Idea, Friend add

### DISCUSSION THREAD

**Discussion by Isamuishi** [vrcx-team/VRCX#473] [Ideas]
So like you know how you can export friends, you should make it to were you can like import your friends and add them all

**Comment by Natsumi-sama** [comment]
This isn't possible because VRChat rate limits spam sending friend requests.

**Reply by Isamuishi** [reply]
but like, isnt it possible to time it out then go again

**Reply by Myrkie** [reply]
that's possible but very unsafe to do, and could possibly get your account flagged, and it has in the past in my testing, adding unsafe features to VRCX is not something that we want to do
